### DISCUSSION TITLE
中央是美食沙漠嗎

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2273] [General]
身為大三生的我，每次到午餐時間都不知道要吃什麼，感覺已經吃到膩了，不過最近到那個什麼碳烤土司吃晚餐，晚餐提供飯類服務，他的碳烤雞腿飯是真的頂，而且免費加飯跟豆芽菜，大家還知道什麼推薦吃飯地點嗎

**Comment by Teddy111802518** [comment]
尤其是我室友又很挑，這個很貴那個不好吃，搞得每次都吃那幾家，自己吃又有點無聊，最近是有再次拜訪櫥窗，東坡肉真的好吃，啊我個人也覺得阿米林偏貴，花露我覺得其實價格還行，我是有竟然說很貴，吃過更便宜的，是我台北住久了高物價看多了嗎，到底哪裡也什麼110的石鍋拌飯

**Comment by Mminniehuang** [comment]
住在南部真的覺得物價又貴又不怎麼好吃...
隨便一間炒飯啊滷味啊都便宜大碗又好吃，尤其是海鮮，原來我從小的吃的海鮮這麼新鮮，自從在這邊的夜市吃了一次烤魷魚就再也不相信北部的海鮮了。
（推薦到南部讀大學
